docs/book/src/reference/metrics.md

```diff
@@ -92,5 +94,26 @@ func init() {
 ```
 
 You may then record metrics to those collectors from any part of your
-reconcile loop, and those metrics will be available for prometheus or
+reconcile loop.
+
+For example:
+```
+func (r *MemcachedReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	...
+	...
+	...
+
+	// Add metrics
+	goobers.Inc()
+	gooberFailures.Inc()
+
+	return ctrl.Result{}, nil
+}
+```
+
+Those metrics will be available for prometheus or
```

Comment:
@VenkatRamaraju Looks good! Can you also add a note that if someone wants to publish metrics on Prometheus UI, they would have to configure their prometheus instance to select the service monitor based on labels.